mcs/class/WindowsBase/ZipSharp/UnzipReadStream.cs

```diff
@@ -30,6 +30,7 @@ namespace zipsharp
 {
 	class UnzipReadStream : Stream
 	{
+		private static bool use32Bit = IntPtr.Size == 4 || Environment.OSVersion.Platform != PlatformID.Unix;
```

Comment:
@marek-safar Maybe just put the `use32Bit` field in IOFunctions? It's used for both zip and unzip...